<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddStudentWindow</class>
 <widget class="QDialog" name="AddStudentWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>283</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>391</width>
     <height>201</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="5" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="sPhotoButton">
        <property name="text">
         <string>Take Photo</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sAddButton">
        <property name="text">
         <string>Add Student</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Student ID :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sSIDLineEdit">
        <property name="placeholderText">
         <string>1234567A</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sFpButton">
        <property name="text">
         <string>Scan Fingerprint</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Student name :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sNameLineEdit">
        <property name="placeholderText">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sClearButton">
        <property name="text">
         <string>Clear</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Email :</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="sEmailLineEdit">
        <property name="text">
         <string/>
        </property>
        <property name="placeholderText">
         <string>example@student.gla.ac.uk</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="sCardButton">
        <property name="text">
         <string>Scan Card</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>240</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
